src/test/java/ToDosTest.java

```diff
@@ -0,0 +1,43 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Tests the functionality of the {@link ToDos} class.
+ * This class verifies the correct behavior of the ToDos task class
+ * regarding its initialization, string representation, and storage format.
+ */
+public class ToDosTest {
+
+    //test setup
+    ToDos todo1 = new ToDos("Read book");
+
+    @Test
+    public void statusInitializeTest1() {
```

Comment:
Should there be header comments for each method to give a short description of the test?